LocalSettings.php

```diff
@@ -2935,8 +2936,9 @@
 	],
 	'wgMFNoindexPages' => [
 		'wmgUseMobileFrontend' => false,
-		'gratisdatawiki' => true,
-		'gratispaideiawiki' => true,
+		'default' => true,
```

Comment:
No. This should not be done..the default is already false, by the wmgUseMobileFrontend part of this. Which makes it only set config if MobileFrontend is enabled.

Reply:
The default is supposed to be true according to extension.json https://github.com/wikimedia/mediawiki-extensions-MobileFrontend/blob/master/extension.json#L870
So what I want is false for my wikis.

Reply:
This does the opposite

Reply:
@RhinosF1 how do you mean?